wiki/Silence/de.md

```diff
@@ -0,0 +1,40 @@
+# Stummschaltung
+
+**Stummschaltung** (auch *Silence* genannt) ist ein Begriff, um die Bestrafung zu beschreiben, bei der einem Spieler die Kommunikation innerhalb der osu!-Community verwehrt wird. Stummschaltungen können durch Mitglieder des [Globalen Moderationsteams](/wiki/People/The_Team/Global_Moderation_Team) und des [Nomination Assessment Teams](/wiki/People/The_Team/Nomination_Assessment_Team) festgelegt werden und werden nur für Spieler aufgestellt, die gegen die [Community-Regeln](/wiki/Rules) verstoßen haben.
+
+## Beschränkungen
+
+::: Infobox
+![](img/silence-profile.png "Eintrag einer Stummschaltung auf der Profilseite")
+:::
+
+Nutzer, die von Moderatoren stumm geschaltet werden, können folgende Dinge nicht machen:
+
+- Den [Chat](/wiki/Client/Interface/Chat_console) benutzen (öffentlich und privat), sowohl im Spiel als auch auf der Webseite
+- In irgendeinem der osu!-Foren beitragen
+- Kommentare auf der Webseite hinterlassen (Neuigkeiten, Änderungsprotokoll, Beatmaps)
+- An [Beatmap-Diskussionen](/wiki/Beatmap_discussion) teilnehmen
+- Profildetails bearbeiten (z. B. der Avatar, die Sektion "me!")
+- Beatmaps einreichen und aktualisieren
+- An [Mehrspieler](/wiki/Multi)-Spielen teilnehmen
+
+Zusätzlich zu dem werden alle Nachrichten eines stumm geschalteten Spielers aus allen Chat-Kanälen und privaten Konversationen entfernt [[1]](https://blog.ppy.sh/post/38114063519/this-week-in-osu-5).
+
+::: Infobox
+![](img/silence-ingame.png "Warnung der Stummschaltung im Chat")
+:::
+
+Nachdem ein Nutzer stumm geschaltet wurde, wird eine Warnung anstelle der Chateingabe angezeigt, die anmerkt, wann die Stummschaltung vorbei ist. Der Eintrag der Stummschaltung kann auch am unteren Ende des Profils auf der Webseite eingesehen werden. Dort wird dieser nach 28 Tagen verschwinden.
+
+## Einer Stummschaltung widersprechen
+
+Wenn du glaubst, mit Unrecht stumm geschaltet worden zu sein, dann kontaktiere das [Account-Support-Team](/wiki/People/The_Team/Account_support_team#accounts@ppy.sh) via [accounts@ppy.sh](mailto:accounts@ppy.sh), um die Stummschaltung anzufechten. Stelle sicher, dass du die mit deinem osu!-Account verknüpfte E-Mail-Adresse verwendest. Erwähne deinen osu!-Benutzernamen sowie das Datum und die ungefähre Zeit, als die Stummschaltung vergeben wurde.
+
+## Häufige Gründe für Stummschaltungen
+
+*Vorsicht: Unangemessenes Verhalten in privaten Nachrichten zu zeigen, ist nicht gegen die Regeln, solange dieses Verhalten nicht bei Leuten verwendet wird, die dies nicht möchten. Sofern dies gemeldet wird, werden entsprechende Strafen angewandt.*
+
+- Unangemessenes Verhalten in öffentlichen Chats oder Foren (z. B. in `#multiplayer`, Beatmap-Diskussionen, Kommentaren, usw.), einschließlich—aber nicht beschränkt auf—Spamming, Unsinn im Chat, Rassismus, etc.
+- Unangemessene Inhalte über andere Wege hochladen oder bereitstellen. (Z. B., Profilavatare, Einreichungen von Beatmaps, Profildetails, Banner, etc.).
```

Comment:
not sure what `user page bio` in the english article should mean anyway, because i thought it would mean the `"me!" section`, but higher up they wrote `profile details (e.g. avatar, "me!" section)`

Reply:
yeah, adding `me! section` is kinda redundant here if it is part of the `user page bio`. Is it okay to just leave it like that?

Reply:
should be fine